devicetypes/smartthings/aeon-led-bulb.src/aeon-led-bulb.groovy

```diff
@@ -182,30 +239,51 @@ def setColor(value) {
 		def rgb = huesatToRGB(hue, saturation)
 		result << zwave.switchColorV3.switchColorSet(red: rgb[0], green: rgb[1], blue: rgb[2], warmWhite:0, coldWhite:0)
 	}
-
-	if(value.hue) sendEvent(name: "hue", value: value.hue)
-	if(value.hex) sendEvent(name: "color", value: value.hex)
-	if(value.switch) sendEvent(name: "switch", value: value.switch)
-	if(value.saturation) sendEvent(name: "saturation", value: value.saturation)
+	result += queryAllColors()
```

Comment:
does this need a delay, in your experience?

Reply:
The commands() call right after it will add a 200ms delay. queryAllColors() just builds a list of switchColorGet commands.

Reply:
👍